src/stats/src/InfoTheory.C

```diff
@@ -22,15 +22,14 @@
 //
 //-----------------------------------------------------------------------el-
 
-#include <queso/InfoTheory.h>
-
 #include <queso/Defines.h>
-#include <gsl/gsl_sf_psi.h> // todo: take specificity of gsl_, i.e., make it general (gsl or boost or etc)
 
 #ifdef QUESO_HAS_ANN
+#include <gsl/gsl_sf_psi.h> // todo: take specificity of gsl_, i.e., make it general (gsl or boost or etc)
```

Comment:
It's good practice to put third-party dependency headers below own-library headers.  Prevents you from accidentally omitting that dependency in any own-library headers that need it.